drafts/use-cases/1.entry-point.md

```diff
@@ -28,18 +28,27 @@ HTTP 200 OK
 ```
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@id": "/api",
     "@type": "hydra:EntryPoint",
-    "events": {
-        "@id": "/api/events", 
+    "#events": {
+        "@id": "/api/events",
         "title": "List of events",
         "@type": "hydra:Collection",
-        "operation": {
-            "@type": "hydra:Operation",
-            "title": "Create new event",
-            "method": "POST",
-            "expects": "schema:Event"
-        }
+        "operation": [
+            {
+               "@type": ["hydra:Operation", "schema:ViewAction"],
+                "title": "List events",
+                "method": "GET",
```

Comment:
> Ie. if GET operation is absent, one could assume that GET is not supported, right?

No, Hydra is based on the open world assumption. Unless something is explicitly marked as being forbidden you can't draw such a conclusion.

> you may have resources which are write-only. How would you model that?

Currently that isn't possible. The client would need to figure that out at runtime. Do we have such a use case?